adjust mnemonic popup: change from placement='right' to 'right end'

kiley-mitti opened the pull request:
Made this change and a couple of small styling changes. The placement change is because setting it to just 'right' doesn't seem to allow it to adjust upward and downward but setting it to right-end or right-start will make the poopup adjust so that it is never flows off the screen. Also changed the trigger to inline-block because inline display wasn't allowing the arrow the space it needed.

Paterma reviewed (approved):
This looks great. 👍🏼 While you're in Mnemonic land, want to change the font to interactive and cursor to pointer for the threshold input column? Doesn't have to be in this PR, just remembered when we chatted about it.

kiley-mitti commented:
@Paterma that's a great idea. I've added that in. Changed it to a button which gives pretty much everything we want plus I think it's semantically more correct since clicking it triggers an action. Check it out!

NicDominguez commented on src/Components/Watcher/LineChart.tsx:
```diff
@@ -27,6 +27,9 @@ const LineChart = ({ data, chartData }: PropTypes) => {
       toolbar: {
         show: false,
       },
+      zoom: {
+        enabled: false,
+      },
```
Good catch, thank you.

NicDominguez reviewed (approved):
Looks much better and good choice to use the secondary button.
